Makefile

```diff
@@ -22,6 +22,21 @@ NVCC_CUDNN =
 # By default we don't build with cudnn because it blows up compile time from a few seconds to ~minute
 USE_CUDNN ?= 0
 
+# We will place .o files in the `build` directory (create it if it doesn't exist)
+BUILD_DIR = build
+ifeq ($(OS), Windows_NT)
```

Comment:
Delete line 27 - line 38. I don't think we need to create the directory. I tested on Windows and Ubuntu 24. And the path separator for Windows can be / here in the Makefile. Tested on both OS'.

Reply:
Optionally - you can remove line 26 as well and just replace BUILD_DIR with build in the two instances below.